libs/ui/jest.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     ...shared.collectCoverageFrom,
     '!src/ContestTally.tsx',
     '!src/LogoMark.tsx',
+    '!src/NumberPad.tsx',
```

Comment:
You may remove this after this is merged: https://github.com/votingworks/vxsuite/pull/838